M/MUMPS/build_tarballs.jl

```diff
@@ -17,22 +17,35 @@ mkdir -p ${libdir}
 cd $WORKSPACE/srcdir/MUMPS_5.2.1
 
 OPENBLAS=(-lopenblas)
+FFLAGS=()
+CFLAGS=()
 if [[ ${nbits} == 64 ]] && [[ ${target} != aarch64* ]]; then
+  FFLAGS+=(-ffixed-line-length-none)  # replacing symbols below sometimes makes lines > 72 chars
   OPENBLAS=(-lopenblas64_)
+  if [[ "${target}" == powerpc64le-linux-gnu ]]; then
+    OPENBLAS+=(-lgomp)
+  fi
+
+  syms=(DGEMM IDAMAX ISAMAX SNRM2 XERBLA ccopy cgemm cgemv cgeru cher clarfg cscal cswap ctrsm ctrsv cungqr cunmqr dcopy dgemm dgemv dger dlamch dlarfg dorgqr dormqr dnrm2 dscal dswap dtrsm dtrsv dznrm2 idamax isamax ilaenv scnrm2 scopy sgemm sgemv sger slamch slarfg sorgqr sormqr snrm2 sscal sswap strsm strsv xerbla zcopy zgemm zgemv zgeru zlarfg zscal zswap ztrsm ztrsv zungqr zunmqr)
+  for sym in ${syms[@]}
+  do
+    FFLAGS+=("-D${sym}=${sym}_64")
+    CFLAGS+=("-D${sym}=${sym}_64")
+  done
 fi
 
 makefile="Makefile.G95.PAR"
 cp Make.inc/${makefile} Makefile.inc
 
 make_args+=(OPTF=-O
             CDEFS=-DAdd_
-            LMETISDIR=/workspace/destdir/lib
-            IMETIS=/workspace/destdir/include
-            LMETIS="-lparmetis -lmetis"
+            LMETISDIR=${libdir}
+            IMETIS=-I${prefix}/include
+            LMETIS='-L$(LMETISDIR) -lparmetis -lmetis'
```

Comment:
Are you sure this line does what you expect?  Within single quotes variables are not expanded, and `$(METISDIR)` is the syntax for command substitution, not variable expansion

Reply:
It's variable expansion in the Makefile. I pass it with single quotes so that the `LMETIS` variable defined in the Makefile takes a value based on the variable `LMETISDIR`, also defined in the Makefile. I supposed I could have used double quotes and `${METISDIR}`. The result is the same.

Reply:
> I supposed I could have used double quotes and ${METISDIR}. The result is the same.

Yes, this is what I would have done, but I confirm that it looks like the Makefile does the expansion correctly